sqlstore/src/main/resources/db/migration/V8510__libraryTemplate_volumeUnits.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE LibraryTemplate ADD COLUMN volumeUnits varchar(30);
+UPDATE LibraryTemplate SET volumeUnits = 'MICROLITRES' WHERE defaultVolume IS NOT NULL;
```

Comment:
Can you help me understand what this is doing? I'm confused about why it's changing existing volumeUnits to `MICROLITRES`, and leave NULL volumeUnits alone (and it looks like that's intended, from the lack of default on the volumeUnits column).

Reply:
If default volume is set, then volume units should also be set. Microlitres is the only volume unit that currently exists in MISO, so I'm setting that on all the templates that already specify a volume

Reply:
Makes sense, thanks!